.vib/redmine/cypress/cypress/fixtures/smtps.json

```diff
@@ -0,0 +1,7 @@
+{
+  "newSMTP": {
+    "smtpEmailAddress": "redmine@vmware.net",
```

Comment:
I'd rather use a truly fake one, just in case. e.g. `redmine@example.com`

Reply:
Why, what is the difference?

Reply:
I am usually not very into using real domains, just in case haha but feel free to leave it as it is. It was just a personal preference